docs/tape.md

```diff
@@ -0,0 +1,59 @@
+# [tape](https://github.com/substack/tape)
+
+## Usage
+
+```js
+// using ES6 modules
+import test from 'tape'
+import { assertions } from 'redux-actions-assertions'
+
+// using CommonJS modules
+var test = require('tape')
+var assertions = require('redux-actions-assertions');
```

Comment:
I guess we have a mistake here in js docs, should be:
var assertions = require('redux-actions-assertions').assertions;
Needs to be changed also in ja version